packages/hw-app-ada/src/Ada.js

```diff
@@ -16,31 +16,18 @@
  ********************************************************************************/
 // @flow
 
-import Int64 from "node-int64";
 import type Transport from "@ledgerhq/hw-transport";
 import { TransportStatusError } from "@ledgerhq/hw-transport";
 
-const CLA = 0x80;
+const CLA = 0xD7;
+const HARDENED = 0x80000000;
 
-const INS_GET_PUBLIC_KEY = 0x01;
-const INS_SET_TX = 0x02;
-const INS_SIGN_TX = 0x03;
-const INS_APP_INFO = 0x04;
-
-const P1_FIRST = 0x01;
-const P1_NEXT = 0x02;
-const P1_LAST = 0x03;
-
-const P2_SINGLE_TX = 0x01;
-const P2_MULTI_TX = 0x02;
-
-const MAX_APDU_SIZE = 64;
-const OFFSET_CDATA = 5;
-const MAX_ADDR_PRINT_LENGTH = 12;
-const INDEX_MAX = 0xffffffff;
+const INS_GET_VERSION = 0x00;
+const INS_GET_EXT_PUBLIC_KEY = 0x10;
 
+const INVALID_PATH = 0x5001;
```

Comment:
Are these from ledger or they are pure JS exception values. I guess we should distinguish them somehow (for now at least by comment)